apis/quay/v1/quayregistry_types.go

```diff
@@ -150,6 +152,7 @@ const (
 	ComponentMirrorReady        ConditionType = "ComponentMirrorReady"
 	ComponentMonitoringReady    ConditionType = "ComponentMonitoringReady"
 	ComponentTLSReady           ConditionType = "ComponentTLSReady"
+	ComponenntConfigEditorReady ConditionType = "ComponentConfigEditorReady"
```

Comment:
you have a typo in the const name, replace `ComponenntConfigEditorReady` with `ComponentConfigEditorReady`.
furthermore, where is this condition being used? maybe we need something in `pkg/cmpstatus` to controll it?